src/omotes_rest/db_models/job_rest.py

```diff
@@ -25,7 +26,7 @@ class JobRest(Base):
     """Name of the workflow this job runs."""
     status: JobRestStatus = db.Column(db.Enum(JobRestStatus), nullable=False)
     """Last received status of the job."""
-    progress_fraction: float = db.Column(db.Float, nullable=False)
+    progress_fraction: Mapped[float] = mapped_column(db.Float, nullable=False)
```

Comment:
Apparently there is a newer way of defining columns: https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html#using-annotated-declarative-table-type-annotated-forms-for-mapped-column